flink-ml-parent/flink-ml-lib/pom.xml

```diff
@@ -57,4 +57,30 @@ under the License.
 			<version>1.1.2</version>
 		</dependency>
 	</dependencies>
+
+	<build>
+		<plugins>
+			<!-- Build flink-ml-lib jar -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>shade-flink</id>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+						<configuration>
+							<artifactSet>
+								<includes combine.children="append">
+									<include>com.github.fommil.netlib:core</include>
```

Comment:
licensing needs to be updated

Reply:
Hi, I'm not sure if I get your point. The license of `com.github.fommil.netlib:core` has already been included?

Reply:
@zentol cc

Reply:
Why was the license already included if the dependency wasn't bundled so far?

Reply:
@zentol The license has already been added by this commit https://github.com/apache/flink/pull/8631/files

Or do you mean we should remove the licensing for release-1.10 and release-1.9? I think we can create a fix in another PR.

Reply:
If the dependency is not bundled in 1.10 then yes, the licensing part of the commit should be reverted.

Reply:
hmm. in fact the flink-ml-* packages are never release/bundled anyway. we should revert the LICENSE notice (in fact it shouldn't contain any LICENSE since only source codes were released: https://repo1.maven.org/maven2/org/apache/flink/flink-ml-lib_2.12/1.10.0/.

How should we proceed? should we have a revert in release-1.10 branch only and keep this PR intact?

Reply:
@walterddr Hi, flink-ml-* are released by jar artifacts and the flink-ml-lib jar should not contain the license and notice file as it actually does not bundle the related dependencies. I have fired another issue([FLINK-16241](https://issues.apache.org/jira/browse/FLINK-16241)) and will open a corresponding PR to address the problem on release-1.10. :-)